From fc971da1cabd32218bc36856890e96f8b294b0e5 Mon Sep 17 00:00:00 2001
From: Hu Kejun <william.hu@rock-chips.com>
Date: Thu, 6 Aug 2020 11:53:00 +0800
Subject: [PATCH] ARM: dts: rv1126: add
 rkcif_dvp/rkcif_mipi_lvds/rkcif_lite_mipi_lvds node

Signed-off-by: Hu Kejun <william.hu@rock-chips.com>
Change-Id: I31bcd3442256af0aa7e1cc7ab859d3abca05899c
---
 arch/arm/boot/dts/rv1126.dtsi | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/arch/arm/boot/dts/rv1126.dtsi b/arch/arm/boot/dts/rv1126.dtsi
index 83fa1b830551..d7eee0d47aa8 100644
--- a/arch/arm/boot/dts/rv1126.dtsi
+++ b/arch/arm/boot/dts/rv1126.dtsi
@@ -330,6 +330,27 @@
 		};
 	};
 
+	rkcif_dvp: rkcif_dvp {
+		compatible = "rockchip,rkcif-dvp";
+		rockchip,hw = <&rkcif>;
+		iommus = <&rkcif_mmu>;
+		status = "disabled";
+	};
+
+	rkcif_lite_mipi_lvds: rkcif_lite_mipi_lvds {
+		compatible = "rockchip,rkcif-mipi-lvds";
+		rockchip,hw = <&rkcif_lite>;
+		iommus = <&rkcif_lite_mmu>;
+		status = "disabled";
+	};
+
+	rkcif_mipi_lvds: rkcif_mipi_lvds {
+		compatible = "rockchip,rkcif-mipi-lvds";
+		rockchip,hw = <&rkcif>;
+		iommus = <&rkcif_mmu>;
+		status = "disabled";
+	};
+
 	rockchip_suspend: rockchip-suspend {
 		compatible = "rockchip,pm-rv1126";
 		status = "disabled";
-- 
2.35.3

